app/controllers/pages_controller.rb

```diff
@@ -8,9 +8,9 @@ def home; end
   def index
     @incomes = Income.first(5)
     @expenses = Expense.first(5)
-    @expenses_categories = Category.joins(:expenses).group('title').sum('expenses.value')
-    current_expenses_value = Expense.sum(:value)
-    current_budget_value = Budget.last.value
+    @expenses_categories = Category.joins(:expenses).group('title').sum('expenses.value_cents')
+    current_expenses_value = Expense.sum(:value_cents)
+    current_budget_value = Budget.last.value_cents
```

Comment:
Since you added the association between the user, it probably should be `current_user.budgets.last.value_cents` :grin: